frontend/src/components/misc/KowlTable.tsx

```diff
@@ -560,15 +560,14 @@ export class SearchTitle extends Component<{
         setTimeout(this.focusInput);
 
         return <span>
-            <span >{this.props.title}</span>
-            <div className="tableInlineSearchBox"
+            {!props.observableFilterOpen.filterOpen && <span>{this.props.title}</span>}
```

Comment:
small suggestion, `<Text>` (not a big deal)

Reply:
I actually wanted to discuss this before. Are we sure about migrating stuff to Chakra that doesn't actually provide any value. I'd personally vote to sticking to standard HTML as much as possible.

We're now using Chakra and will use it for some time, but hypothetically if in a future it's decided to migrate away from it to something else that ever changing JS world will provide, I think it'll be easier to migrate away from using HTML rather than Charka. I know we're not migrating away from Chakra, but who knows what will be usable and valuable in 2-3 years timeframe.

Of course, if there is any value in using `Box` instead of `div`, `Text` instead of `p` that I'm not seeing, I'm all up to it.

WDYT?